src/cleaning/cleaning.c

```diff
@@ -107,6 +108,10 @@ static int _ocf_cleaner_run_check_dirty_inactive(ocf_cache_t cache)
 static void ocf_cleaner_run_complete(ocf_cleaner_t cleaner, uint32_t interval)
 {
 	ocf_cache_t cache = ocf_cleaner_get_cache(cleaner);
+	ocf_queue_t iter;
+
+	list_for_each_entry(iter, &cache->io_queues, list)
+		ocf_engine_reschedule_deferred(iter);
```

Comment:
I know it is not required but could you place this line in brackets? I think this would improve readability.